sources-themes.ts

```diff
@@ -113,6 +113,18 @@ export const sourcesCommunity: ThemeSource[] = [
     source: 'https://github.com/akamud/vscode-theme-onelight/blob/master/themes/OneLight.json',
     type: 'light',
   },
+  {
+    name: 'plastic-deprioritised-punctuation',
```

Comment:
The name of this theme feels a bit too long - do you think we could come up with a better name, or we only have the main version?

Reply:
I thought something similar. I thought of maybe using `plastic-dp`, but that acronym has some... unsavoury connotations, depening on who you ask 😅. I figured I would leave that call up to a maintainer to decide if that's too risky or not.

I also considered something like `plastic-alt`. That'd would work well, especially since `alt` is fairly self-explanatory when looking at the list of names. The only downside is that it would differ from what the official extension calls it. Other variants include something like `plastic-alt-punct`, `plastic-deprio`, `plastic-depunct`, or some other short mismash of the two words 😄.

Reply:
I don't use that variation personally, no. I included both because I have no idea how many other users prefer it. If they were from separate VS extensions, there'd be another install count to give us an idea...

I guess since contributing themes here is so a hassle-free, it's not a big deal to leave one out for later. I'll remove the second version for now. 🙂